mlprimitives/jsons/mlprimitives.candidates.dsp.SpectralMask.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "mlprimitives.candidates.dsp.SpectralMask",
+    "author": "Rodrigo Diez <rjdiez@mit.edu>",
+    "documentation": "",
+    "description": "Anomalies detection in satellite telemetry data using a spectral mask",
+    "modalities": [],
+    "primitive": "mlprimitives.candidates.dsp.SpectralMask",
+    "fit": {
+        "method": "fit",        
+        "args": [
+            {
+                "name": "X",
+                "description": "Training vector",
+                "type": "ndarray"
+            }
+        ]
+    },
+
+    "produce": {
+        "method": "produce",
+        "args": [
+            {
+                "name": "X",
+                "description": "Telemetry data",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                
+                "name": "anomalies",
+                "description": "Data vector consisting of the anomalies detected in the telemetry data",
+                "type": "ndarray"
+            }
+        ]
+    },
+
+    "hyperparameters": {
+        "tunable": {
+            "method": {

```

Comment:
I'm not sure about it, but should this rather be a "fixed" hyperparameter?
If it's not, and it's something that we expect to modify during the tuning process (if it actually makes sense to try multiple options) you should add a list of possible values: `"values": ["std_dev", "min_max"]`

Reply:
ok, it was not clear to me the difference between both, it's more clear now in the light of the tuning process. Thanks! I will change it and I will add "values" anyway.